plugins/hookup.py

```diff
@@ -23,13 +23,17 @@ def hookup(db, conn, chan):
         return "something went wrong"
     person1 = people[random.randint(0,len(people) - 1)]
```

Comment:
Perhaps using `random.shuffle(people)` and just grabbing the first two elements would help guarantee a user won't be matched with themselves. In that case a length check would probably also be required.